base/src/main/java/io/spine/validate/MessageFieldValidator.java

```diff
@@ -67,11 +61,10 @@ protected void validateOwnRules() {
             validateFields();
             BuiltInValidation.ANY.validateIfApplies(this);
         }
-        BuiltInValidation.TIMESTAMP.validateIfApplies(this);
     }
 
     private boolean shouldValidateFields() {
-        return getValidateOption() && !fieldValueNotSet();
+        return fieldValue().valueOf(OptionsProto.valid) && !fieldValueNotSet();
```

Comment:
Please extract variables for each of the component. It would make it clearer what we check. This is especially important since we have double negation at the end of the expression.